packages/next/src/build/index.ts

```diff
@@ -2983,12 +2983,13 @@ export default async function build(
                 if (i18n) {
                   if (additionalSsgFile) return
 
+                  const localeExt = page === '/' ? path.extname(file) : ''
```

Comment:
what is this lol

Reply:
The `path.extname()` doesn't have to be called inside the loop as it doesn't change